### ISSUE TITLE
DependencyError: 'CommonActQuant' can not resolve attribute 'is_narrow_range'

### ISSUE THREAD

**Issue by dustinjoe** [Xilinx/brevitas#228]
Run into this error when running that BNN cifar10 training script command(full error msg shown as below):

model, _ = model_with_cfg('cnv_2w2a', pretrained=True)

-------------------------
Traceback (most recent call last):

  File "/media/xyzhou/extDisk2t/_CVPR21/cifar10/torchadvtoolbox0.py", line 60, in <module>
    model, _ = model_with_cfg('cnv_2w2a', pretrained=True)

  File "/media/xyzhou/extDisk2t/_CVPR21/cifar10/utils.py", line 67, in model_with_cfg
    model = model_impl[arch](cfg)

  File "/media/xyzhou/extDisk2t/_CVPR21/cifar10/CNV.py", line 127, in cnv
    in_ch=in_channels)

  File "/media/xyzhou/extDisk2t/_CVPR21/cifar10/CNV.py", line 55, in __init__
    restrict_scaling_type=RestrictValueType.POWER_OF_TWO))

  File "/home/xyzhou/anaconda3/lib/python3.7/site-packages/brevitas/nn/quant_activation.py", line 137, in __init__
    **kwargs)

  File "/home/xyzhou/anaconda3/lib/python3.7/site-packages/brevitas/nn/quant_layer.py", line 88, in __init__
    **kwargs)

  File "/home/xyzhou/anaconda3/lib/python3.7/site-packages/brevitas/nn/mixin/act.py", line 193, in __init__
    **kwargs)

  File "/home/xyzhou/anaconda3/lib/python3.7/site-packages/brevitas/nn/mixin/act.py", line 88, in __init__
    assert isinstance(act_quant, ActQuantProxyProtocol)

  File "/home/xyzhou/anaconda3/lib/python3.7/site-packages/typing_extensions.py", line 1380, in __instancecheck__
    for attr in _get_protocol_attrs(cls)):

  File "/home/xyzhou/anaconda3/lib/python3.7/site-packages/typing_extensions.py", line 1380, in <genexpr>
    for attr in _get_protocol_attrs(cls)):

  File "/home/xyzhou/anaconda3/lib/python3.7/site-packages/_dependencies/injector.py", line 68, in __getattr__
    raise DependencyError(message)

DependencyError: 'CommonActQuant' can not resolve attribute 'is_narrow_range'
------------------------------------------------

Thank you!

**Comment by volcacius** [comment]
Hello,

Thanks for reporting this. On which commit of brevitas are you? I can't reproduce it here, so maybe it's something that has been fixed in more recent versions.

**Comment by dustinjoe** [comment]
Thanks for your reply. Let me try to reinstall and further check it. I guess this might also be caused by other packages.

**Comment by dustinjoe** [comment]
Hello. 
I actually solved this issue later. It is because in that 'common.py' file the import dependencies got changed in the recent version of Brevitas but I did not notice this in my running. Actually these two first lines:

from dependencies import value
from brevitas.inject import BaseInjector as Injector

Two more little questions
1. Is there a way I can train an original floating point model for this CNV cifar10 model architecture? I want to make some performance comparisons on it?
2. Is there a performance estimation statistics tool inside that can show me the computation complexity of the model? For example in Google's qKeras their model summary return the number of MACs for a model?

Thank you!
